crates/orderbook/openapi.yml

```
@@ -579,11 +579,16 @@ components:
         Provides the additional data for ethflow orders
       type: object
       properties:
-        is_refunded:
+        refunded_status:
```

nit:
```suggestion
        refundedStatus:
```

---

```suggestion
        automaticRefundStatus:
```

I suggested a bit above to either call this `automatic_refund_status` and track the state of the automatic refunds, or this state should then track both the automatic/manual refunds. What are we planning to track here?